ginga/util/heaptimer.py

```diff
@@ -26,13 +26,12 @@
 __version__ = '1.0'
```

Comment:
Note: Unrelated to this PR but I notice that this file has Apache License. Do we need to declare this in `ginga/licenses`?